website/addons/onedrive/client.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+import logging
+
+#import requests #TODO: remove this after determining onedrive connection issues w/make_request
+
+from requests_oauthlib import OAuth2Session
+from oauthlib.oauth2 import InvalidGrantError
+
+from framework.exceptions import HTTPError
+
+from website.util.client import BaseClient
+#from website.addons.googledrive import settings
+from website.addons.googledrive import exceptions
```

Comment:
Maybe (some of) these exceptions could be generalized and added to `website.addons.base.execptions`?